### PR TITLE
Add high block rate and connections alert

### PR DESCRIPTION
**Pull request by CodeSandwich**
Fixes https://github.com/radicle-dev/infra/issues/92 and https://github.com/radicle-dev/infra/issues/93

### CODE REVIEW

File: registry/ffnet/grafana_alerts_dashboard.json [hunk 1]
```diff
@@ -173,22 +175,45 @@
             "query": {
               "params": [
                 "A",
-                "1h",
+                "10s",
```
Comment (geigerzaehler) [inline registry/ffnet/grafana_alerts_dashboard.json:178]: I believe this is what causes the alert spam with `Error: tsdb.HandleRequest() error unmarshalerDecoder`.

Using 10 seconds as a value here means that the alert is based on the data from the last 10 seconds. If an alert rule is evaluated it is conceivable that there is no data available for the last 10 seconds. (Prometheus scrapes only every 5 seconds and there is a processing delay.) I suggest we increase this to 1 minute and use this values for all alerts.

File: registry/ffnet/grafana_alerts.md [hunk 2]
```diff
@@ -10,42 +10,61 @@ It's not applied automatically during creation of a Grafana instance.
...
+## Node is not connected to its peers
 ### Name
-`Block import rate 11m low`
+`Peer connections low`
 ### Goal
-Check if blocks are being mined and that the chain is growing
+Check if the nodes in connected to at least validator nodes and the RPC server.
+### Trigger
+All the nodes are checked every 10 seconds for their `substrate_sub_libp2p_peers_count` reading.
+If this reading in a 10 second window is below threshold, the alert is risen.
```
Comment (geigerzaehler) [inline registry/ffnet/grafana_alerts.md:32]: I think we can be a bit more lenient and say that only if the number of peers is below the threshold for a minute we alert.

Comment (CodeSandwich) [reply registry/ffnet/grafana_alerts.md:32]: Fixed

File: registry/ffnet/grafana_alerts_dashboard.json (hunk 1 above)
Comment (CodeSandwich) [reply registry/ffnet/grafana_alerts_dashboard.json:178]: Good idea, definitely worth a try! Done.

File: registry/ffnet/grafana_alerts.md [hunk 3]
```diff
@@ -10,42 +10,61 @@ It's not applied automatically during creation of a Grafana instance.
...
+The thresholds depend on a node's `kubernetes_pod_label_app` label:
+- `miner` - 3
+- `validator` or `rpc-server` - 2
```
[the remark this answers is not in the document]
Comment (CodeSandwich) [reply registry/ffnet/grafana_alerts.md:35]: Added explanation.

**Pull request merged**